docs/guides/admin/docs/modules/awss3archive.md

```diff
@@ -46,20 +46,48 @@ The Opencast AWS S3 Archive service configuration can be found in the
 Using S3 Archiving
 ------------------
 
-There are two major methods to access S3 archiving features: manually, and via a workflow.  Amazon S3 archiving is not
-part of the default workflows and manual S3 offload is disabled by default.  To enable manual S3 offload you must edit
-the `offload.xml` workflow configuration file and change `var s3Enabled = false;` to `var s3Enabled = true;`.  To
-manually offload a media package follow the directions in the user documentation.
+S3 archiving is done on a Snapshot level, that is a mediapackage ID + version.  Because of the way that the Asset
+Manager handles snapshots, all newly created snapshots are *always* local.  Creating a snapshot of a mediapackage with
+non local data will download *all* related snapshots for that mediapackage which can incur significant costs.  S3
+archiving is meant to be a cost reduction, and storage expansion tool, rather than then primary storage for your
+asset manager.  Therefore, most adopters do not want to immediately (ie, at the end of your default workflow) offload
+your recordings to S3!  Instead, we suggest using the TimedMediaArchiver
+(org.opencastproject.assetmanager.impl.TimedMediaArchiver.cfg) to offload your recordings after sufficient time that
```

Comment:
Is this picked up by the JavaScript config linking bit? Does that need `etc/` as prefix?

Reply:
It's not.  But I've reworked that bit, since it didn't read nicely.